File: packages/java/parser-jvm-plugin-backbone/src/test/java/com/vaadin/fusion/parser/plugins/backbone/multiendpoints/MultiEndpointsBarEndpoint.java
package com.vaadin.fusion.parser.plugins.backbone.multiendpoints;

@Endpoint
public class MultiEndpointsBarEndpoint {
    public String getBar() {
        return "bar";
    }

    public MultiEndpointsSharedModel getShared() {
        return new MultiEndpointsSharedModel();
    }
}

File: packages/java/typescript-generator/src/test/resources/com/vaadin/hilla/parser/plugins/backbone/multiendpoints/snapshots/MultiEndpointsBarEndpoint.ts
import { EndpointRequestInit } from "@vaadin/hilla-frontend";
import type MultiEndpointsSharedModel from "./com/vaadin/hilla/parser/plugins/backbone/multiendpoints/MultiEndpointsSharedModel.js";
import client from "./connect-client.default.js";
async function getBar(init?: EndpointRequestInit): Promise<string | undefined> { return client.call("MultiEndpointsBarEndpoint", "getBar", {}, init); }
async function getShared(init?: EndpointRequestInit): Promise<MultiEndpointsSharedModel | undefined> { return client.call("MultiEndpointsBarEndpoint", "getShared", {}, init); }
export { getBar, getShared };
